exercises/04-integration/after/Divergent.Customers/EndpointConfig.cs

```diff
@@ -1,23 +0,0 @@
-
-namespace Divergent.Customers
-{
-    using NServiceBus;
-
-    public class EndpointConfig : IConfigureThisEndpoint
-    {
-        public void Customize(EndpointConfiguration endpointConfiguration)
-        {
-            //TODO: NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence.
-            // Refer to the documentation for more details on specific options.
-            endpointConfiguration.UsePersistence<PLEASE_SELECT_ONE>();
-
-            // NServiceBus will move messages that fail repeatedly to a separate "error" queue. We recommend
-            // that you start with a shared error queue for all your endpoints for easy integration with ServiceControl.
-            endpointConfiguration.SendFailedMessagesTo("error");
-
-            // NServiceBus will store a copy of each successfully process message in a separate "audit" queue. We recommend
-            // that you start with a shared audit queue for all your endpoints for easy integration with ServiceControl.
-            endpointConfiguration.AuditProcessedMessagesTo("audit");
-        }
-    }
-}
```

Comment:
Was this a redundant file? I.e. not included in the project?

Reply:
Not sure, I'll check.

Reply:
yes, it was added by the host package and I accidentally committed it

Reply:
OK, please keep this change as a separate commit.

Reply:
I introduced this file while I was doing the updates AFAIK, so I squashed it...

Reply:
This file has to be present in master right now, otherwise it wouldn't show as a removal in this PR.

Reply:
Yes you are right! It is on master...

What do you think we should do?

Reply:
As discussed, I'm OK to mix this minor change with the package update.